system/box64.box64rc

```diff
@@ -758,6 +758,9 @@ BOX64_DYNAREC_STRONGMEM=1
 BOX64_DYNAREC_SAFEFLAGS=2
 BOX64_DYNAREC_CALLRET=0
 
+[StarRail.exe]
```

Comment:
Would be better to keep alphabetical.

Reply:
Ah looks like the alphabetical order has been disrupted already, never mind.

Reply:
Both StarRail and Zenless Zone Zero is an mihoyo game. I thought it was categorized by game producers. I'll change it

Reply:
> Ah looks like the alphabetical order has been disrupted already, never mind.

Okay then i will keep it here ~ XD